proposals/Level-1.md

```diff
@@ -78,94 +77,69 @@ instruction. Thrown exceptions are handled as follows:
+associated with a try block defines how to handle the throw.
 
-An `event` is an internal construct in WebAssembly that is maintained by the
-embedder. WebAssembly events are defined by a new `event section` of a
-WebAssembly module.
+An `event` (such as an exception) is an internal construct in WebAssembly that
+is maintained by embedder. WebAssembly events are defined by a new `event`
+section of a WebAssembly module. The event section is a list of event types
+that are defined by the module.
 
-The event section defines a list of event types that may be handled by the
-module.  Each `event type` has (at a minimum) an `event specifier` and a `type
-signature`. The event specifier defines the kind of event while the type
-signature defines the list of values associated with the event.  Depending on
+Each `event type` has a `event specifier` and a `type signature`. Depending on
 the value of the event specifier, additional values may be specified in the
-event type.
 
 For level 1, the only event specifier is the constant 0, and denotes that the
-event is an `exception`. It has no additional values specified (other than the
-type signature). Future extensions my add additional event specifiers, such as
-resumable events.
-
-The `type signature` of an event type is an index into the `type` section.  The
-type signature is defined by a function type. The parameter list of the type
-signature defines the list of values returned to the corresponding enclosing
-catch. The result type defines the list of values to be used when resuming on an
-event. For exceptions, that can't resume, the result type must always be type
-'void'.
-
-Note: Resumable events may not use the `throw` instruction. Rather, they may use
-a new (yet to be defined) `yeild` instruction that passes value(s) to its
-enclosing contexts. Such an instruction would allow the calling context to
-return values back, and resume with the instruction following the yeild
-instruction, based on the values returned.
+event type corresponds to an exception. Future extensions my add additional
+event specifiers.
 
-Within a module, event types are identified by an `event index` to the [event
-index space](#event-index-space). Each event index refers to a corresponding
-runtime tag that uniquely identifies the corresponding event to be handled.  The
-corresponding runtime tag is called an `event tag`.
-
-Event types are numbered numerically, starting with the import section, and then
-followed by event types in the event section.  Each event type within a module
-are associated with unique event tags.
+The type signature of an event type is an index to the corresponding function
+signature defined in the `type` section. The parameter of the function signature
+defines the list of values associated with the event. The return value(s) of the
+signature defines what is passed back to yeilding instruction so that execution
+may resume.
```

Comment:
I think for now it'd be better to say that events cannot have a return value. I don't think we want resumable exceptions in the Level 1 proposal, even though we are trying to leave room for them.